frontend/src/helpers/comparsion/find-empty-option.ts

```diff
@@ -0,0 +1,17 @@
+import { ComparisonGeneralInform } from '@autoline/shared';
+
+const findEmptyOptions = (
+  generalInfo: ComparisonGeneralInform[] | undefined,
+): string[] | undefined => {
+  const emptyOptions = generalInfo?.map((e) =>
+    Object.entries(e.options)
+      .filter((option) => option[1].length == 0)
+      .map((option) => option[0]),
+  );
```

Comment:
It's a great practice to destruct the `Object.entries` result instead of using the indices because they are unclear.
`.filter(([smthA, smthB]) => {})`